api/src/data_migration/transformation/transform_oracle_data_task.py

```diff
@@ -253,35 +289,31 @@ def process_assistance_listing(
         target_assistance_listing: OpportunityAssistanceListing | None,
         opportunity: Opportunity | None,
     ) -> None:
-        self.increment(self.Metrics.TOTAL_RECORDS_PROCESSED)
+        self.increment(self.Metrics.TOTAL_RECORDS_PROCESSED, prefix=ASSISTANCE_LISTING)
         extra = {
             "opportunity_assistance_listing_id": source_assistance_listing.opp_cfda_id,
             "opportunity_id": source_assistance_listing.opportunity_id,
         }
         logger.info("Processing assistance listing", extra=extra)
 
-        if opportunity is None:
+        if source_assistance_listing.is_deleted:
+            self._handle_delete(
+                source_assistance_listing, target_assistance_listing, ASSISTANCE_LISTING, extra
+            )
```

Comment:
I don't understand these lines. It looks like it's saying

```
# if source listing is already deleted
# delete it again
```

?

Reply:
Ah nevermind, I just needed to read the PR description ^^

I feel like `is_deleted` means something more like `to_be_deleted` or `marked_for_deletion` ? idk. That's out of scope for this PR though.

Reply:
Yeah, the column is `is_deleted` which makes sense in the context of the table (it _is_ or _isn't_ deleted), but in the context of processing those records does read a bit weird. I could alias the field name using a python property, but that might be more confusing as you'd need to trace where it comes from.